cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -65,8 +65,8 @@ type ScaleUpRequest struct {
 
 // ScaleDownRequest contains information about the requested node deletion.
 type ScaleDownRequest struct {
-	// NodeName is the name of the node to be deleted.
-	NodeName string
+	// Node is the node to be deleted.
+	Node *apiv1.Node
```

Comment:
Do we need to use `Node` instead of `NodeName`?

Reply:
I use `Node` instead of `NodeName` to delete `scaleDownRequest` if there's no instance on the cloud provider side. If we use `NodeName`, I have to get the node object from lister before calling the `HasInstance` function.